### ISSUE TITLE
Rocks floating above ground near battery river

### ISSUE THREAD

**Issue by tedbarnett** [TimeWalkOrg/Manhattan#36]
Walking along the river at ground level reveals a number of rock piles that seem to be too high in the Z axis!

![image](https://github.com/TimeWalkOrg/Manhattan/assets/694013/18ef5870-bac3-421b-af41-fb0a8538b5cd)

**Comment by deedox** [comment]
Yeah that can happen because they are brushed and aligned to the terrain normal.. Easy to fix with smoothing the terrain.

**Comment by tedbarnett** [comment]
How do we do that?
